### ISSUE TITLE
Stuck with low-gas token transfer, unable to boost

### ISSUE THREAD

**Issue by drortirosh** [trezor/trezor-suite#7214]
**Describe the bug**
I tried to make a token transfer directly through the "suite.trezor.io" (instead of metamask)
- submitted transaction was "Legacy" and with relatively low gas price, and prices were rising, so transaction stuck
- I wanted to boost the gas - but the [pending] transaction doesn't appear anywhere in the "suite" UI.
- I wanted to boost it through metamask - but it gets "transaction underpriced" error, since Metamask attempt to re-submit as Type-2 transaction, and even if I specify high maxGas and high PrioFee, it doesn't replace the legacy TX.
(tiny bug of metamask: doesn't show the "underpriced" error)

- bottom line: I'm stuck, until the gas price come down (or the TX is dropped from mempool)

**Info:**

-   Remembered Wallet: yes/no
-   Suite Version: web 22.11.1
-   Browser: chome
-   OS: osx
-   Bridge: version
-   Firmware Version: Trezor T, 1.11.1

**Comment by sime** [comment]
This is fixed in the most recent release 🥳
